gandalf.js

```diff
@@ -10,51 +10,51 @@
  * Released under the MIT license.
```

Comment:
SCRIPT-Consider extracting this repeated logic into a separate function to avoid duplication.